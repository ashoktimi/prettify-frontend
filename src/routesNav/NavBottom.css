.NavBtm{
    height: 20rem;
    margin: 2rem 10% 15px;
    object-fit: cover;
    background-image: url('../images/bottom.jpg');
    padding: 15px;
}