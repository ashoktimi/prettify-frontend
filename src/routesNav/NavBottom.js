import React from "react";
import 'bootstrap/dist/css/bootstrap.css';
import Navbar from 'react-bootstrap/Navbar';
import './NavBottom.css';
const NavbarBottom = () =>{
    return(
        <Navbar className="NavBtm"/>
    )
}

export default NavbarBottom;