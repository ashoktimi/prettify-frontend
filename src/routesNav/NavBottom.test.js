import React from "react";
import { render } from "@testing-library/react";
import NavBottom from './NavBottom';
import { MemoryRouter } from "react-router";

it("matches snapshot", function () {
  const { asFragment } = render(
      <MemoryRouter>
        <NavBottom />
      </MemoryRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
