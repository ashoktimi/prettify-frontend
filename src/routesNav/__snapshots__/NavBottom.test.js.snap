// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <nav
    class="NavBtm navbar navbar-expand navbar-light"
  />
</DocumentFragment>
`;
